.github/workflows/super-linter.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: All in One Linter
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+jobs:
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: symlink
+        run: ln -sf .yamllint.yml .github/linters/.yaml-lint.yml
+
+      - name: Lint Code Base
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_ALL_CODEBASE: true
+          VALIDATE_GO: true  # This will enable golanci linter
+          VALIDATE_YAML: true # This will enable yaml lint
+          VALIDATE_PYTHON_PYLINT: true # This will enable python pylint
+          VALIDATE_BASH_EXEC: true # This will return error if shell script is not executable
+          VALIDATE_GITHUB_ACTIONS	: true # This will enable linter for GitHub action
+          VALIDATE_MARKDOWN: true # This will enable linter for markdown file
+
+          YAML_CONFIG_FILE: .yamllint # Filename for Yamllint configuration
```

Comment:
won't this need complete symlink name, that is, `.yamllint.yml` ?

Reply:
right, I'm trying to resolve this issue 
``` 2021-08-05 07:10:48 [DEBUG]     -> Codebase does NOT have file:[/github/workspace/.yamllint], nor the file:[], using Default rules at:[/action/lib/.automation/.yamllint]
2021-08-05 07:10:48 [DEBUG]     -> Language rules file variable (YAML_LINTER_RULES) value is:[/action/lib/.automation/.yamllint]
2021-08-05 07:10:48 [FATAL]     -> YAML_LINTER_RULES rules file (/action/lib/.automation/.yamllint) doesn't exist. Terminating...
```
still trying will come back to this soon